src/main/java/com/boardgo/domain/user/service/UserCommandServiceV1.java

```diff
@@ -63,4 +68,44 @@ private void validateNickNameAndPrTag(String nickName, List<String> prTags) {
             validatePrTag(prTags);
         }
     }
+
+    @Override
+    public void updateProfileImage(Long userId, MultipartFile profileImage) {
+        UserInfoEntity userInfoEntity = getUserInfoEntity(userId);
+        String originalImage = userInfoEntity.getProfileImage();
+        String newImage = "";
+        if (!profileImage.isEmpty()) {
+            newImage =
+                    s3Service.upload(USER, FileUtils.getUniqueFileName(profileImage), profileImage);
+        }
+        if (existString(originalImage)) {
+            s3Service.deleteFile(originalImage);
+        }
+        userInfoEntity.updateProfileImage(newImage);
+    }
+
+    @Override
+    public void updatePersonalInfo(Long userId, UserPersonalInfoUpdateRequest updateRequest) {
+        UserInfoEntity userInfoEntity = getUserInfoEntity(userId);
+        if (existString(updateRequest.nickName()) && validateNickname(updateRequest.nickName())) {
+            userInfoEntity.updateNickname(updateRequest.nickName());
+        }
+        if (existString(updateRequest.password())) {
+            userInfoEntity.encodePassword(passwordEncoder);
+        }
+    }
+
+    @Override
+    public void updatePrTags(List<String> changedPrTag, Long userId) {
+        List<UserPrTagEntity> prTags = userPrTagRepository.findByUserInfoId(userId);
+        userPrTagRepository.deleteAllInBatch(prTags);
+        validatePrTag(changedPrTag);
```

Comment:
```suggestion
        validatePrTag(changedPrTag);
        userPrTagRepository.deleteAllInBatch(prTags);
```
이거 두개 순서 바꿔야할 것 같아요!